README.md

```diff
@@ -56,7 +56,7 @@ Clone the repository, checkout the tag you want to build, or pick the main branc
 
 ### Install Playbook
 
-* [`playbooks/playbook.yml`](./playbooks/playbook.yml) deploys based on the collections defaults.
+* [`playbooks/amq_streams.yml`](./playbooks/amq_streams.yml) deploys based on the collections defaults.
```

Comment:
I'm confused about the renaming of the files. In the collection the file is playbooks/playbook.yml, not amq_streams.yml.
Do want to rename to amq_streams.yml? It's bit a tricky, because, as a convention, all our collection have a playbooks/playbook.yml that run "everything" on one target (and it's used by our molecule test and our CI).

Reply:
Sorry! It was my fault! Renamed again and the CI pipeline is working successfully!